art/step/steps.py

```diff
@@ -165,7 +168,6 @@ def do(self, previous_states: Dict):
         Args:
             previous_states (Dict): previous states
         """
-        self.model.turn_on_model_regularizations()
```

Comment:
Why did you delete this?

Reply:
We don't have access to the model here now. I think this should be rethough as Regularize step will be developed.